client/extensions/talk-to-plugin/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection } from 'my-sites/controller';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import Main from 'components/main';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import QueryPlugin from './query-plugin';
+import { getSelectedSite } from 'state/ui/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+const render = ( context ) => {
+	renderWithReduxStore( (
+		<LeWidget />
+	), document.getElementById( 'primary' ), context.store );
+};
+
+const Widget = props => {
```

Comment:
Though I personally don't love it, we should use parens around the argument. In this particular case, destructuring may be more convenient: `Widget = ( { isJetpackSite, site } )`